### PR TITLE
2021.10 release highlights

### PR DESCRIPTION
**Pull request by mcarrano**
This adds release highlights for 2021.10 including md file and images.

### CODE REVIEW

File: release-highlights/release-highlights-2021.10/release-highlights.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# PatternFly Release Highlights
+## Release 2021.10
+----------------------------------------------------------
+## New features and enhancements
+
+### [Password strength demo](https://www.patternfly.org/v4/demos/password-strength)
+
+  ![password strength demo](./img/password-strength.png)
+
+Adds a demo to show how to the new Helper text component to add password validation and a strength indicator to a password field.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.10/release-highlights.md:10]:
```suggestion
Adds a demo to show how to use the new Helper text component to add password validation and a strength indicator to a password field.
```

File: release-highlights/release-highlights-2021.10/release-highlights.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+### [Disabled dropdown tooltip](https://www.patternfly.org/v4/components/dropdown#basic)
+
+![tolltip on disabled dropdown item](./img/dropdown-tooltip.png)
+
+Allows a tooltip to be attached to a disabled dropdown menu item.
+
+### [Log viewer enhancements](https://www.patternfly.org/v4/extensions/log-viewer)
+
+The log viewer extension has been updated to support a dark mode and full screen viewing. Note that log viewer has been moved from the Components to the new Expensions area on the PatternFly website.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.10/release-highlights.md:32]:
```suggestion
The log viewer extension has been updated to support a dark mode and full screen viewing. Note that log viewer has been moved from the Components to the new Extensions area on the PatternFly website.
```

File: release-highlights/release-highlights-2021.10/release-highlights.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+* [Horizontal table scrolling with sticky columns](https://github.com/patternfly/patternfly/issues/4218) - Allow a table to scroll horizontally while keeping some number of columns always visible.
+
+* [Back to top component](https://github.com/patternfly/patternfly/issues/4255) - Introduce a new component that floats on top of content for long scrolling pages and navigates the user back to the top of the page in a single click.
+
+* [Plain alert variant](https://github.com/patternfly/patternfly/issues/4220) - Adds a new variant of an inline alert that removes borders and background styling.
+
+* [Expandable inline alert](https://github.com/patternfly/patternfly/issues/4056) - Adds a new alert type with an expandable header that allows body content to be hidden for a more compact appearance.
+
+* [Disabled tabs](https://github.com/patternfly/patternfly/issues/4257) - Add a disabled variant for the tabs component. The use can place a tooltip on a disabled tab as they would for other disabled controls.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.10/release-highlights.md:66]:
```suggestion
* [Disabled tabs](https://github.com/patternfly/patternfly/issues/4257) - Add a disabled variant for the tabs component. The user can place a tooltip on a disabled tab as they would for other disabled controls. 
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Just a few typos otherwise looks good 👍

**Review by nicolethoen — APPROVED** [review]
Is it worth pointing out the new code editors in the workspace? - Like the work Zack did to see the code for each example?

**Comment by mcarrano** [comment]
@evwilkin @nicolethoen thanks for the review. I accepted all of your suggested changes and also added content to highlight the updates to the website including the new code editor and addition of the Extensions section. Please review again.

**Review by evwilkin — APPROVED** [review]
LGTM thanks!

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**